scripts/format_all.sh

```diff
@@ -4,7 +4,9 @@ set -e
 
 cd "$(dirname "$0")/.."
 
-for cmd in clang-format find grep python3 sed xargs; do
+CLANG_FORMAT_BIN="${CLANG_FORMAT_BIN:-clang-format}"
```

Comment:
XXX_BIN often refers to the xxx/bin dir.
For the executable itself, use XXX or XXX_EXECUTABLE.